src/app/api/routes/devices.py

```diff
@@ -57,29 +64,20 @@ async def update_device(payload: DeviceIn, device_id: int = Path(..., gt=0)):
     """
     Based on a device_id, updates information about the specified device
     """
-    if payload.login is not None:
-        updated_device = await crud.get(device_id, devices)
-        if updated_device is not None and payload.login != updated_device["login"]:
-            await check_for_access_login_existence(payload.login)
-            updated_acccess = await crud.fetch_one(accesses, {"login": updated_device["login"]})
-            await update_access_login(payload.login, updated_acccess["id"])
-
-    return await crud.update_entry(devices, payload, device_id)
+    return await crud.accesses.update_accessed_entry(devices, accesses, device_id, payload)
 
 
 @router.delete("/{device_id}/", response_model=DeviceOut, summary="Delete a specific device")
 async def delete_device(device_id: int = Path(..., gt=0), _=Security(get_current_user, scopes=["admin"])):
     """
     Based on a device_id, deletes the specified device
     """
-    entry = await crud.delete_entry(devices, device_id)
-    # Delete access
-    await crud.delete_entry(accesses, entry['access_id'])
-    return entry
+    return await crud.accesses.delete_accessed_entry(devices, accesses, device_id)
 
 
-@router.get("/my-devices", response_model=List[DeviceOut],
-            summary="Get the list of all devices belonging to the current user")
+@router.get(
+    "/my-devices", response_model=List[DeviceOut], summary="Get the list of all devices belonging to the current user"
+)
```

Comment:
I don't really get which formatter you use :sweat_smile:  Sometimes it looks like black, sometimes it doesn't haha

Reply:
I think that I reformatted the file with black then I rollbacked some of the functions I had not modified to make the review easier. This one look like black as it often turn the code into 
```
def/call function(
   many, many, many, parameters, here
)
``` 

instead of something like this for some IDEs
```
def/call function(many, many, many
                  parameters, here)
```

I do prefer the 1st formatting 😉